wrapper/src/main/resources/messages.properties

```diff
@@ -162,7 +162,7 @@ HostSelector.noHostsMatchingRole=No hosts were found matching the requested ''{0
 IamAuthConnectionPlugin.unsupportedHostname=Unsupported AWS hostname {0}. Amazon domain name in format *.AWS-Region.rds.amazonaws.com or *.rds.AWS-Region.amazonaws.com.cn is expected.
 IamAuthConnectionPlugin.useCachedIamToken=Use cached IAM token = ''{0}''
 IamAuthConnectionPlugin.generatedNewIamToken=Generated new IAM token = ''{0}''
-IamAuthConnectionPlugin.invalidPort=Port number: {0} is not valid. Port number should be greater than zero.
+IamAuthConnectionPlugin.invalidPort=Port number: {0} is not valid. Port number should be greater than zero. Falling back to host or dialect port.
```

Comment:
Falling back to default port.
?

Reply:
Just changed the messaging to "...Falling back to default port."